### ISSUE TITLE
How to config database in my home directory when funannotate installed globally on the server in a read-only central location

### ISSUE THREAD

**Issue by Jennie17** [nextgenusfs/funannotate#103]
Dear nextgenusfs,

Our HPC server administrator has installed funannotate globally in a read-only central location.

Under my home dir, I downloaded all the databases as required.

However, when I started the test run from my project dir, I always got the error msg: database is not configured properly.

Could you please advise how to configure the database in my home directory, so that funannotate installed globally can work properly?

Looking forward to your reply

Best wishes

**Comment by nextgenusfs** [comment]
Hi @Jennie17.  Right now funannotate uses a symlink in the 'DB' folder in the install directory.  So the error message is suggesting that the symlink is not properly setup, i.e. it is unable to find the database files. If you don't have write access to that install folder, then the script is likely unable to generate the symlink. I have on my list of things to update how the database is built and stored, however, I don't know when I will get to this and will require an updated funannotate installation.  Probably the simplest thing would be to allow for a `--database` option at runtime where you can specify where your files are located.  Sorry this isn't working, I will try to get to it as soon as possible.  I developed funannotate with a single CPU in mind, so there are some growing pains moving to an HPC.  In the meantime, you could also try to run the Docker version if that is an option?

**Comment by Jennie17** [comment]
Hi Nextgenusfs,

Many thanks for your answer and looking forward to the development of --database option.

Alternatively, will it be possible to run funannotate on a desktop?

If using a normal desktop or laptop, how many hours are generally needed for the annotation of a genome 100-200 Mb?

Does funannotate require root access for installation? If not, then I can try to install the whole thing under my folder, then it won't have the writing access issue.

Looking forward to your reply

Best wishes

**Comment by Jennie17** [comment]
Hi Nextgenusfs,

Regarding the Docker version you mentioned, do you refer to the setup method on the website below?
https://github.com/nextgenusfs/funannotate/wiki/funannotate-on-Docker

For this alternative approach, shall I be able to install it in my project folder on HPC without worrying about the root access. etc?

Thanks again!

**Comment by nextgenusfs** [comment]
@Jennie17 you shouldn't need root access for installation, just the tools need to be available in your $PATH, but you should be able to control that in your environment.  If all of the other tools are properly installed system wide, you should just be able to clone the repository or download a release and then add the /funannotate/ folder to your path.

The length of time for a run depend on how much evidence you have for the annotation, whether you let it train Augustus, etc.  Typically for a smaller fungal genome (~30 Mb) `funannotate predict` should finish in under 12 hours with ~ 6 CPUs. 

If your HPC allows you to run Docker - that might be an option (I've heard that many administrators do not allow docker containers).

I'll update this thread when a new version is released that has better database handling.

**Comment by Jennie17** [comment]
Hi Nextgenusfs,

Many thanks for your help.

At the end are all the dependencies installed system wide.

I have also git cloned funannotate into my working directory: /scratch/WLR53/funannotate

Could you please describe what I need to do to install funannoate under my folder /scratch/WLR53?

I tried to follow this link, but obviously most steps won't fit in the current situation.
https://github.com/nextgenusfs/funannotate/wiki/Ubuntu-Installation

Looking forward to your reply

Thanks again!

Checking External Dependencies....
-----------------------------------------------
blastp installed........./usr/local/blast+/2.6.0/bin/blastp
hmmsearch installed........./usr/local/hmmer/3.1b2/bin/hmmsearch
hmmscan installed........./usr/local/hmmer/3.1b2/bin/hmmscan
augustus installed........./usr/local/augustus/3.2.1/bin/augustus
gmes_petap.pl installed........./usr/local/genemark-es/4.21/gmes_petap.pl
mummer installed........./usr/local/mummer3/3.23/mummer
nucmer installed........./usr/local/mummer3/3.23/nucmer
show-coords installed........./usr/local/mummer3/3.23/show-coords
exonerate installed........./usr/local/exonerate/2.2.0/bin/exonerate
gmap installed........./usr/local/gmap/2017-06-20/bin/gmap
blat installed........./usr/local/blat/3.6/blat
RepeatModeler installed........./usr/local/repeatmodeler/1.0.8/RepeatModeler
RepeatMasker installed........./usr/local/repeatmasker/4.0.6/RepeatMasker
pslCDnaFilter installed........./usr/local/funannotate/0.7.2/bin/pslCDnaFilter
bedtools installed........./usr/local/bedtools/2.25.0/bin/bedtools
bamtools installed........./usr/local/bamtools/2.4.0/bin/bamtools
gag.py installed........./usr/local/funannotate/0.7.2/bin/gag.py
tbl2asn installed........./usr/local/tbl2asn/bin/tbl2asn
braker.pl installed........./usr/local/braker1/1.9/braker.pl
funannotate installed........./usr/local/funannotate/0.7.2/funannotate
mafft installed........./usr/local/mafft/7.300/bin/mafft
trimal installed........./usr/local/trimal/1.4.1/trimal
raxmlHPC-PTHREADS installed........./usr/local/raxml/8.2/bin/raxmlHPC-PTHREADS
tRNAscan-SE installed........./usr/local/tRNAscan-SE/1.3.1/bin/tRNAscan-SE
rmOutToGFF3.pl installed........./usr/local/repeatmasker/4.0.6/util/rmOutToGFF3.pl
proteinortho5.pl installed........./usr/local/proteinortho/v5.16/proteinortho5.pl
-----------------------------------------------
Checking Python modules....
-----------------------------------------------
biopython installed
psutil installed
natsort installed
goatools installed
sklearn installed
matplotlib installed
seaborn installed
numpy installed
pandas installed
-----------------------------------------------
Checking Perl modules....
-----------------------------------------------
Bio::SeqIO installed
Pod::Usage installed
File::Basename installed
threads installed
threads::shared installed
Thread::Queue installed
Getopt::Long installed
FindBin installed
File::Spec installed
File::Path installed
Data::Dumper installed
YAML installed
Carp installed
Hash::Merge installed
Logger::Simple installed
Parallel::ForkManager installed
-----------------------------------------------

**Comment by nextgenusfs** [comment]
You should just be able to install the databases , i.e. run 
```
/scratch/WLR53/funannotate/funannotate setup -m all -d /scratch/WLR53/funannotate_database
```

**Comment by Jennie17** [comment]
[jiwu2573@login2 funannotate]$ ./funannotate setup -m all -d /scratch/WLR53/funannotate_database
Traceback (most recent call last):
  File "./funannotate", line 2, in <module>
    import funannotate
  File "/scratch/RDS-FSC-WLR53-RW/funannotate/funannotate.py", line 6, in <module>
    from natsort import natsorted
ImportError: No module named natsort

**Comment by nextgenusfs** [comment]
pip install --user natsort

**Comment by nextgenusfs** [comment]
strangely the python environment isn't picking up the modules?

**Comment by Jennie17** [comment]
First
module load funannotate (load the system wide installed funannotate first to include all the required dependencies)
then 
./funannotate setup -m all -d /scratch/WLR53/funannotate_database
it's downloading

Can this way work out?

**Comment by nextgenusfs** [comment]
Not sure, it might.  I'm not familiar with HPCs and the PATH environment, seems odd that the system wide python modules wouldn't be available.

**Comment by Jennie17** [comment]
I think I shall first do 
module load python
that could be the reason

**Comment by Jennie17** [comment]
If this step works, then I shall be able to run the command below straight away?

$ funannotate predict -i genome1.fasta -o genome1 -s "Genome one" \
    --isolate fun1 --name GN01_ --augustus_species botrytis_cinerea \
    --protein_evidence proteins.fa --transcript_evidence transcripts.fa --cpus 6

**Comment by nextgenusfs** [comment]
perhaps, but you may need to either put the local copy of funannotate in your path ahead of the system wide install one, otherwise it may default to the one that is "broken"

**Comment by Jennie17** [comment]
Hi Nextgenusfs,

Thanks for your help.

It looks like the test run is getting through.

For using the sample data provided, is the predict_results folder under genome1 expected to be empty (see pasted at the end)?

[jiwu2573@login2 WLR53]$ cat funanno.e1622104
[01:17:58 PM]: OS: linux2, 24 cores, ~ 132 GB RAM. Python: 2.7.9
[01:17:59 PM]: Running funannotate v0.7.2
[01:17:59 PM]: Downloading dikarya busco models
[01:25:13 PM]: Augustus training set for botrytis_cinerea already exists, thus funannotate will use those parameters.
  If you want to re-train, provide a unique name for the --augustus_species argument
[01:25:14 PM]: AUGUSTUS (3.2.1) detected, version seems to be compatible with BRAKER1 and BUSCO
[01:25:14 PM]: Loading sequences and soft-masking genome
[01:25:14 PM]: Soft-masking: building RepeatModeler database
[01:25:20 PM]: Soft-masking: generating repeat library using RepeatModeler
[01:26:00 PM]: Soft-masking: running RepeatMasker with custom library
[01:26:32 PM]: Masked genome: 3 scaffolds; 451,050 bp; 0.81% repeats masked
[01:26:32 PM]: Aligning transcript evidence to genome with GMAP
[01:27:58 PM]: 220 transcripts aligned with GMAP
[01:27:58 PM]: Aligning transcript evidence to genome with BLAT
[01:27:59 PM]: 469 filtered BLAT alignments
[01:27:59 PM]: Mapping proteins to genome using tBlastn/Exonerate
[01:28:00 PM]: Using 9,043 proteins as queries
[01:28:00 PM]: Running pre-filter tBLASTN step
[01:28:04 PM]: Found 3 preliminary alignments
[01:28:06 PM]: Exonerate finished: found 0 alignments
[01:28:06 PM]: Running GeneMark-ES on assembly
[01:29:37 PM]: Converting GeneMark GTF file to GFF3
[01:29:37 PM]: Found 113 gene models
[01:29:37 PM]: Running Augustus gene prediction
[01:30:30 PM]: Found 186 gene models
[01:30:30 PM]: Pulling out high quality Augustus predictions
[01:30:30 PM]: Found 68 high quality predictions from Augustus (>90% exon evidence)
[01:30:30 PM]: 381 total gene models from all sources
[01:30:31 PM]: Setting up EVM partitions
[01:30:31 PM]: Generating EVM command list
[01:30:31 PM]: Running EVM commands with 5 CPUs
[01:30:42 PM]: Combining partitioned EVM outputs
[01:30:42 PM]: Converting EVM output to GFF3
[01:30:43 PM]: Collecting all EVM results
[01:30:43 PM]: 183 total gene models from EVM
[01:30:43 PM]: Predicting tRNAs
[01:30:45 PM]: Found 1 tRNA gene models
[01:30:45 PM]: Merging EVM output with tRNAscan output
[01:30:45 PM]: Reformatting GFF file using GAG
Traceback (most recent call last):
  **File "/scratch/RDS-FSC-WLR53-RW/funannotate/bin/funannotate-predict.py", line 1050, in <module>
    lib.GAGprotClean(GAG_proteins_original, GAG_proteins)
  File "/scratch/RDS-FSC-WLR53-RW/funannotate/lib/library.py", line 1505, in GAGprotClean
    with open(input, 'ru') as infile:
IOError: [Errno 2] No such file or directory: 'genome1/predict_misc/gag1/genome.proteins.fasta'**


[jiwu2573@login2 genome1]$ ls * -lht
predict_misc:
total 21M
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 154K Sep 15 13:30 evm.trnascan.gff
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW    0 Sep 15 13:30 gag1.proteins.fasta
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  289 Sep 15 13:30 trnascan.gff3
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  238 Sep 15 13:30 tRNAscan.out
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 154K Sep 15 13:30 evm.round1.gff3
drwxr-sr-x 5 jiwu2573 RDS-FSC-WLR53-RW 4.0K Sep 15 13:30 EVM
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 137K Sep 15 13:30 augustus.evm.gff3
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 282K Sep 15 13:30 augustus.gff3
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  43K Sep 15 13:30 augustus-HiQ.evm.gff3
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 260K Sep 15 13:30 gene_predictions.gff3
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 260K Sep 15 13:30 gene_predictions.gff3.tmp
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  125 Sep 15 13:30 weights.evm.txt
drwxr-sr-x 6 jiwu2573 RDS-FSC-WLR53-RW 4.0K Sep 15 13:29 genemark
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  82K Sep 15 13:29 genemark.evm.gff3
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  84K Sep 15 13:29 genemark.gff
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  82K Sep 15 13:29 genemark.temp.gff
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 940K Sep 15 13:29 gmhmm.mod
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW    0 Sep 15 13:28 exonerate.out
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  67K Sep 15 13:28 hints.PE.gff
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW    0 Sep 15 13:28 hints.P.gff
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  224 Sep 15 13:28 p2g.tblastn.out
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW    0 Sep 15 13:28 protein_alignments.gff3
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  51K Sep 15 13:27 blat.filt.psl
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 112K Sep 15 13:27 blat.psl
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  51K Sep 15 13:27 blat.sort.psl
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  51K Sep 15 13:27 blat.sort.tmp.psl
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  67K Sep 15 13:27 hints.E.gff
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 4.1M Sep 15 13:27 proteins.combined.fa
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 287K Sep 15 13:27 gmap-map.log
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  61K Sep 15 13:27 transcript_alignments.gff3
drwxr-sr-x 3 jiwu2573 RDS-FSC-WLR53-RW 4.0K Sep 15 13:27 genome
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 5.5K Sep 15 13:27 gmap-build.log
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 450K Sep 15 13:26 genome.softmasked.fa
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  18K Sep 15 13:26 repeatmasker.gff3
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  12M Sep 15 13:26 transcripts.combined.fa
drwxr-sr-x 2 jiwu2573 RDS-FSC-WLR53-RW 4.0K Sep 15 13:26 RepeatMasker
drwxr-sr-x 3 jiwu2573 RDS-FSC-WLR53-RW 4.0K Sep 15 13:26 RepeatModeler
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 1.4K Sep 15 13:26 repeatmodeler.lib.fa
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 441K Sep 15 13:25 genome.fasta
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW   35 Sep 15 13:25 scaffold.sort.order.txt
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW   42 Sep 15 13:25 scaffold.sort.rename.txt

logfiles:
total 132K
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 106K Sep 15 13:30 funannotate-predict.log
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 7.2K Sep 15 13:30 funannotate-EVM.log
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 1.4K Sep 15 13:30 augustus-parallel.log
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 1.7K Sep 15 13:28 funannotate-p2g.log
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 6.4K Sep 15 13:26 funannotate-repeats.log

**predict_results:
total 0**

**Comment by nextgenusfs** [comment]
What version of GAG do you have installed? i.e. `gag.py -v`?  Seems like that could be the problem.

**Comment by Jennie17** [comment]
[jiwu2573@login3 WLR53]$ module load funannotate
[jiwu2573@login3 WLR53]$ gag.py -v
Traceback (most recent call last):
  File "/usr/local/funannotate/0.7.2/bin/gag.py", line 7, in <module>
    from src.controller import Controller
ImportError: No module named src.controller

**Comment by nextgenusfs** [comment]
Seems like that is the problem, perhaps in the HPC system install as well. https://github.com/genomeannotation/GAG is a requirement, make sure it is installed an working.

**Comment by Jennie17** [comment]
[jiwu2573@login3 logfiles]$ cat funannotate-predict.log
...
2017-09-15 13:30:45,259: Found 1 tRNA gene models
2017-09-15 13:30:45,259: Merging EVM output with tRNAscan output
2017-09-15 13:30:45,262: Reformatting GFF file using GAG
2017-09-15 13:30:45,263: gag.py -f /scratch/RDS-FSC-WLR53-RW/genome1/predict_misc/genome.softmasked.fa -g genome1/predict_misc/evm.trnascan.gff -o genome1/predict_misc/gag1 --fix_start_stop
2017-09-15 13:30:45,332: Traceback (most recent call last):
  File "/usr/local/funannotate/0.7.2/bin/gag.py", line 7, in <module>
    from src.controller import Controller
ImportError: No module named src.controller

**Comment by Jennie17** [comment]
Hi nextgenusfs,

Many thanks for your help.

I found the difference in the bin folder between HPC central location and my own project folder.

It looks like my project folder has much less *.py than the central location.

On HPC central location:
[jiwu2573@login3 logfiles]$ ls /usr/local/funannotate/0.7.2/bin
activate              f2py                           funannotate-fix.py         funannotate-sort_rename.py  pip               python2          wheel
activate.csh          fetch_associations.py          funannotate-functional.py  gag.py                      pip2              python2.7        write_hierarchy.py
activate.fish         fetch_associations.pyc         funannotate-p2g2.py        map_to_slim.py              pip2.7            python-config    write_hierarchy.pyc
activate_this.py      find_enrichment.py             funannotate-p2g.py         map_to_slim.pyc             plot_go_term.py   runxlrd.py
augustus_parallel.py  find_enrichment.pyc            funannotate-predict.py     natsort                     plot_go_term.pyc  runxlrd.pyc
easy_install          funannotate-compare.py         funannotate-remote.py      nosetests                   pslCDnaFilter     vba_extract.py
easy_install-2.7      funannotate-contig_cleaner.py  funannotate-runEVM.py      nosetests-2.7               python            vba_extract.pyc

In my project folder:
[jiwu2573@login3 WLR53]$ ls /scratch/WLR53/funannotate/bin
augustus_parallel.py    funannotate-contig_cleaner.py  funannotate-functional.py  funannotate-p2g.py      funannotate-remote.py  funannotate-sort_rename.py
funannotate-compare.py  funannotate-fix.py             funannotate-p2g2.py        funannotate-predict.py  funannotate-runEVM.py

My project installation was set up using the command as you suggested previously:
/scratch/WLR53/funannotate/funannotate setup -m all -d /scratch/WLR53/funannotate_database

When I ran the test, I use the commands:
fundir=/scratch/WLR53/funannotate/sample_data

module load funannotate
module load augustus

cd $PBS_O_WORKDIR

#run funannotate predict on genome 1
/scratch/WLR53/funannotate/funannotate predict -i ${fundir}/genome1.fasta -o genome1 -s "Genome one" \
    --isolate fun1 --name GN01_ --augustus_species botrytis_cinerea \
    --protein_evidence ${fundir}/proteins.fa --transcript_evidence ${fundir}/transcripts.fa --cpus 6

**Comment by nextgenusfs** [comment]
The error is not a database error, GAG is not properly installed.

**Comment by Jennie17** [comment]
Anyway, I will install https://github.com/genomeannotation/GAG in my project folder first and add it to my $PATH, see how it goes.

**Comment by Jennie17** [comment]
[jiwu2573@login3 GAG]$ python gag.py -v
GAG v2.0.1

**Comment by nextgenusfs** [comment]
as long as it is in $PATH, retry the test data run.

**Comment by Jennie17** [comment]
Still the same error although I added it to the $PATH
2017-09-16 12:19:58,223: Traceback (most recent call last):
  File "/usr/local/funannotate/0.7.2/bin/gag.py", line 7, in <module>
    from src.controller import Controller
ImportError: No module named src.controller

fundir=/scratch/WLR53/funannotate/sample_data
**export PATH=/scratch/WLR53/GAG:$PATH**


module load funannotate
module load augustus

cd $PBS_O_WORKDIR

#run funannotate predict on genome 1
/scratch/WLR53/funannotate/funannotate predict -i ${fundir}/genome1.fasta -o genome1b -s "Genome one" \
    --isolate fun1 --name GN01_ --augustus_species botrytis_cinerea \
    --protein_evidence ${fundir}/proteins.fa --transcript_evidence ${fundir}/transcripts.fa --cpus 6

**Comment by nextgenusfs** [comment]
what does `which gag.py` return? is it the incorrectly installed system version?

**Comment by Jennie17** [comment]
[jiwu2573@login3 WLR53]$ which gag.py
/scratch/WLR53/GAG/gag.py

**Comment by Jennie17** [comment]
[jiwu2573@login3 WLR53]$ gag.py -v
GAG v2.0.1

**Comment by Jennie17** [comment]
However, I have just checked the system module, GAG is not installed.

It's only available in my project dir.

**Comment by Jennie17** [comment]
Hi nextgenusfs,

After GAG is also installed in the system, I seem to get a completed set of results pasted at the end.

Is this what supposed to be?

I can try the 2nd command now:
$ funannotate annotate -i genome1 -e youremail@domain.edu --cpus 6

genome1d/predict_results:
total 1.5M
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  31K Sep 22 15:59 genome_one.discrepency.report.txt
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  165 Sep 22 15:59 genome_one.error.summary.txt
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 694K Sep 22 15:59 genome_one.gbk
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 109K Sep 22 15:59 genome_one.gff3
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  48K Sep 22 15:59 genome_one.proteins.fa
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 441K Sep 22 15:59 genome_one.scaffolds.fa
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  48K Sep 22 15:59 genome_one.tbl
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 139K Sep 22 15:59 genome_one.transcripts.fa
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 2.7K Sep 22 15:59 genome_one.validation.txt

genome1d/predict_misc:
total 21M
drwxr-sr-x 2 jiwu2573 RDS-FSC-WLR53-RW 4.0K Sep 22 15:59 tbl2asn
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  97K Sep 22 15:59 ncbi.cleaned.gff3
drwxr-sr-x 2 jiwu2573 RDS-FSC-WLR53-RW 4.0K Sep 22 15:59 gag2
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 9.3K Sep 22 15:59 mapping.ids
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  98K Sep 22 15:59 ncbi.cleaned.gff3.bak
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  44K Sep 22 15:59 bad_models.gff
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  98K Sep 22 15:59 cleaned.gff3
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 8.7K Sep 22 15:59 genome.repeats.to.remove.gff
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 3.6K Sep 22 15:59 repeat.gene.models.txt
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 274K Sep 22 15:59 repeats.xml
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 152K Sep 22 15:59 evm.trnascan.gff
drwxr-sr-x 2 jiwu2573 RDS-FSC-WLR53-RW 4.0K Sep 22 15:59 gag1
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  87K Sep 22 15:59 gag1.proteins.fasta
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  289 Sep 22 15:59 trnascan.gff3
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  238 Sep 22 15:58 tRNAscan.out
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 152K Sep 22 15:58 evm.round1.gff3
drwxr-sr-x 5 jiwu2573 RDS-FSC-WLR53-RW 4.0K Sep 22 15:58 EVM
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 137K Sep 22 15:58 augustus.evm.gff3
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 282K Sep 22 15:58 augustus.gff3
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  43K Sep 22 15:58 augustus-HiQ.evm.gff3
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 256K Sep 22 15:58 gene_predictions.gff3
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 256K Sep 22 15:58 gene_predictions.gff3.tmp
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  125 Sep 22 15:58 weights.evm.txt
drwxr-sr-x 6 jiwu2573 RDS-FSC-WLR53-RW 4.0K Sep 22 15:58 genemark
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  78K Sep 22 15:58 genemark.evm.gff3
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  81K Sep 22 15:58 genemark.gff
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  78K Sep 22 15:58 genemark.temp.gff
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 940K Sep 22 15:57 gmhmm.mod
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 3.1K Sep 22 15:56 exonerate.out
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  68K Sep 22 15:56 hints.PE.gff
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  521 Sep 22 15:56 hints.P.gff
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  13K Sep 22 15:56 p2g.tblastn.out
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  581 Sep 22 15:56 protein_alignments.gff3
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  51K Sep 22 15:56 blat.filt.psl
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 112K Sep 22 15:56 blat.psl
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  51K Sep 22 15:56 blat.sort.psl
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  51K Sep 22 15:56 blat.sort.tmp.psl
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  67K Sep 22 15:56 hints.E.gff
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 4.1M Sep 22 15:56 proteins.combined.fa
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 287K Sep 22 15:56 gmap-map.log
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  61K Sep 22 15:56 transcript_alignments.gff3
drwxr-sr-x 3 jiwu2573 RDS-FSC-WLR53-RW 4.0K Sep 22 15:56 genome
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 5.5K Sep 22 15:56 gmap-build.log
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 450K Sep 22 15:55 genome.softmasked.fa
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  16K Sep 22 15:55 repeatmasker.gff3
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW  12M Sep 22 15:55 transcripts.combined.fa
drwxr-sr-x 2 jiwu2573 RDS-FSC-WLR53-RW 4.0K Sep 22 15:55 RepeatMasker
drwxr-sr-x 3 jiwu2573 RDS-FSC-WLR53-RW 4.0K Sep 22 15:54 RepeatModeler
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 1001 Sep 22 15:54 repeatmodeler.lib.fa
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 441K Sep 22 15:53 genome.fasta
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW   35 Sep 22 15:53 scaffold.sort.order.txt
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW   42 Sep 22 15:53 scaffold.sort.rename.txt

genome1d/logfiles:
total 36K
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 9.3K Sep 22 15:59 funannotate-predict.log
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 7.2K Sep 22 15:58 funannotate-EVM.log
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 1.4K Sep 22 15:58 augustus-parallel.log
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 1.7K Sep 22 15:56 funannotate-p2g.log
-rw-r--r-- 1 jiwu2573 RDS-FSC-WLR53-RW 5.9K Sep 22 15:55 funannotate-repeats.log

**Comment by Jennie17** [comment]
Hi nextgenusfs,
For the command 
$ funannotate annotate -i genome1 -e youremail@domain.edu --cpus 6

I got the error msg:
funannotate-functional.py: error: unrecognized arguments: -e  ...

Does the email address has to be @mail.edu ?

If I don't have one, can I use my personal email address?

Looking forward to your reply

Thanks!

**Comment by nextgenusfs** [comment]
email address bug has been fixed in v1.0.0
